packages/compass-connections/src/stores/connections-store-redux.ts

```diff
@@ -469,8 +469,17 @@ const INITIAL_STATE: State = {
 };
 
 export function getInitialConnectionsStateForConnectionInfos(
-  connectionInfos: ConnectionInfo[] = []
+  connectionInfos: ConnectionInfo[] | null
```

Comment:
Probably should've been like that from the beginning instead of falling back on empty array and building logic around it, but it's only used in tests so have no real effect on anything until now